public/views/index.html

```diff
@@ -2,7 +2,7 @@
 		<!-- Sidebar -->
 		<div id="sidebar-container" class="sidebar">
 			<div id="logout" class="sidebar" ng-model="user">
-				<a href="/">logout</a> | {{user.email}}
+				<a href="/" ng-click="logout()">logout</a> | {{user.email}}
```

Comment:
`s-ref="logout"` instead? also, since we're redirecting to `/login` when you logout, idk if it makes sense to link to `/` since it'll be redirected anyway